### DISCUSSION TITLE
Discussion about Unlit roads with pavements/sidewalk

### DISCUSSION THREAD

**Discussion by timothywashere** [streetcomplete/StreetComplete#3824] [General]
I walk along country roads in the dark, sometimes I find a road that has a pavement even though it is unlit. 
I really want to map this so that other people can find it easily, it's a lot safer. 
SC currently doesn't show the pavement/sidewalk quest for unlit roads, I believe to avoid "quest spam"
I would really like it to show the quest.
In my experience country roads are the least susceptible to "quest spam" because there's hardly anything around. Most people probably aren't walking on country roads often anyway so are unlikely to be bothered by the extra quests.

@Discostu36 Do you think so also?
https://github.com/streetcomplete/StreetComplete/commit/55888892c997903864157b15f67e2759b49cabec

What do other people think?

**Comment by riQQ** [comment]
How are these "country roads" tagged? Please provide one or more examples.

**Comment by timothywashere** [comment]
Some are primary roads

https://www.openstreetmap.org/way/166255179
https://www.google.com/maps/@50.3443259,-4.6800139,3a,75y,292.45h,81.62t/data=!3m6!1e1!3m4!1swMU0GUGDpkTDW3aFoH4meg!2e0!7i16384!8i8192

Some  minor / unclassified

https://www.openstreetmap.org/way/1035813002
https://www.google.com/maps/@50.4433423,-4.0880012,3a,75y,272.08h,68.73t/data=!3m7!1e1!3m5!1sA_qVGP1yUdpneQoKluZWhA!2e0!6shttps:%2F%2Fstreetviewpixels-pa.googleapis.com%2Fv1%2Fthumbnail%3Fpanoid%3DA_qVGP1yUdpneQoKluZWhA%26cb_client%3Dmaps_sv.tactile.gps%26w%3D203%26h%3D100%26yaw%3D274.47122%26pitch%3D0%26thumbfov%3D100!7i16384!8i8192

**Reply by mnalis** [reply]
I agree, those happen also in Croatia (mostly outside of big cities)

**Comment by Discostu36** [comment]
I don't know why you tagged me and don't really have an opinion on that topic.

**Reply by timothywashere** [reply]
Sorry, I must have misunderstood. You commented on the code change originally. https://github.com/streetcomplete/StreetComplete/commit/55888892c997903864157b15f67e2759b49cabec

**Reply by Discostu36** [reply]
Ah sorry, I didn't remember and the Android app doesn't show commit discussions. I think that at least in urban areas the sidewalk quest should always active for appropriate road types. In rural areas I'm not sure. Here in Germany there are usually no sidewalks outside of cities, only separate paths sometimes.

**Comment by matkoniecz** [comment]
>  I believe to avoid "quest spam"

I expect that it is mostly to avoid asking questions than in vast majority will have answer "no".

One idea: have a separate quest asking about sidewalks on more roads, including unlit, which would be disabled by default. (but that can open gates to doubling quest count)
